### DISCUSSION TITLE
整合包搜索

### DISCUSSION THREAD

**Discussion by kile14134** [Meloong-Git/PCL#4117] [提问] [answered]
感觉整合包搜索功能有问题 在输入完整名称后仍无法直接找到整合 希望可以将搜索功能更新

**Comment by MoYuan-CN** [comment]
提供所搜索内容，结果和预期的整合包

**Reply by kile14134** [reply]
很多很多 我就举一个例子 Create Lost and Renaissance这个包 
![2eb67afb407c6a8178ed2e8344e92bee](https://github.com/Hex-Dragon/PCL2/assets/99176915/e214729e-e9ff-45e5-b72f-bffa7e9eb0a8)
单纯搜是无法在第一页找到的

**Comment by WForst-Breeze** [comment, marked as answer]
本质应与 #228 相同，应该是 CurseForge 的问题 🤔
